code/script.js

```diff
@@ -1,51 +1,246 @@
+    console.log(message)
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/bot-icon.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+    `;
+  };
+  
+  // Makes the chat scroll to the last message when there are too many in the chat box.
   chat.scrollTop = chat.scrollHeight
+};
+
+// Bot pop-up-function.
+// Adds a class to hide ticket-btn-container + removes class from chatbot so it's visible.
+const botPopUp = () => {
+  ticketBtnContainer.classList.add("ticketBtnContainer");
+  chatbot.classList.remove("chatbot");
+  setTimeout(() => greetUser(), 500);
 }
 
-// Starts here
+ticketBtn.addEventListener("click", botPopUp);
+
+// Start of the bot
+// Greeting and Question 1
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
+  // here we call the function showMessage, that we declared earlier with the argument "..." for message, and the argument "bot" for sender.
+  showMessage("Welcome to the Cinema City TicketBot!", 'bot');
+  setTimeout(() => {
+    showMessage("To proceed, I need the name of the person who makes the reservation. Please, enter your name.", 'bot');
+  }, 500);
+  sendBtn.addEventListener('click', handleNameInput);
+};
+
+// Customer name pops up in chat on user's side.
+const handleNameInput = (event) => {
+  event.preventDefault();
+  customerName = inputField.value;
+
+  if (customerName === "") {
+    alert(`You need to write your name.`);
+  } else {
+    showMessage(customerName, "user");
+    clearInputField();
+    // Shows the next question after 1s by calling the next function.
+    setTimeout(() => ticketType(), 1000);
+  };
+};
+
+// Question 2
+// Bot replies back and ask if the ticket are for a adult or child.
+const ticketType = () => {
+  removeChildrenFunction();
+  showMessage(`Nice to meet you ${customerName}! Is the movie ticket for a adult, a student or a child (Under 7)?`, 'bot');
+  // Two buttons for age choices.
+  inputWrapper.innerHTML += `
+    <button id="adult" type="submit">Adult</button>
+    <button id="student" type="submit">Student</button>
+    <button id="child" type="submit">Child</button>
+  `;
+  // Variables that store references to the two added buttons. 
+  const adultBtn = document.getElementById('adult');
+  const studentBtn = document.getElementById('student');
+  const childBtn = document.getElementById('child');
+  //Eventlisteners for the buttons.
+  adultBtn.addEventListener('click', () => {
+    ticketOption = reservation.type[0];
```

Comment:
I can see now why putting all the information into arrays and objects made building the site a whole lot easier! Has made your code a lot more DRY than it would have been otherwise.